tests/cargo-fmt/main.rs

```diff
@@ -1,8 +1,8 @@
 // Integration tests for cargo-fmt.
 
 use std::env;
-use std::process::Command;
 use std::path::Path;
+use std::process::Command;
```

Comment:
Was this meant to be part of the PR?

Reply:
Great question. I think adding a CI check for tests formatting would be great so that we don't run into issues like this in the future!

Reply:
Opened #5212!